e2e/pages/Table/TableModule.ts

```diff
@@ -329,11 +352,62 @@ class TableModule {
       'isCurrentWallet',
     ];
 
-    const okButton = this.page.getByRole('button', { name: OK });
+    const assertionData = {
+      registrationStatus,
+      paId,
+      balance,
+      spentThisMonth,
+      isCurrentWallet,
+    };
+
+    await this.exportAndAssertData(
+      expectedColumns,
+      assertionData,
+      debitCardUsage,
+    );
+  }
+
+  async exportPayMentData({ status, amount }: ExportPaymentAssertionData) {
+    const expectedColumns = [
+      'referenceId',
+      'id',
+      'status',
+      'payment',
+      'timestamp',
+      'registrationStatus',
+      'phoneNumber',
+      'paymentAmountMultiplier',
+      'amount',
+      'financialserviceprovider',
+      'firstName',
+      'lastName',
+      'whatsappPhoneNumber',
+      'addressCity',
+      'addressPostalCode',
+      'addressHouseNumberAddition',
+      'addressHouseNumber',
+      'addressStreet',
+    ];
```

Comment:
This could be placed in the interface on top of the POM. I think... Let me know.

Reply:
Yes, I can do that, then another variable with the same name under exportDebitCardData function need to move to the top too.